pkg/infra/metrics/frontendmetrics.go

```diff
@@ -0,0 +1,43 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// PostPostFrontendMetricsCommand sent by frontend to record frontend metrics
+type PostFrontendMetricsCommand struct {
+	Events []FrontendMetricEvent `json:"events"`
+}
+
+// FrontendMetricEvent a single metric measurement event
+type FrontendMetricEvent struct {
+	Name  string  `json:"name"`
+	Value float64 `json:"value"`
+}
+
+// FrontendMetricsRecorder handles the recording of the event, ie passes it to a prometheus metric
+type FrontendMetricsRecorder func(event FrontendMetricEvent)
+
+// MFrontendLoadTime is a metric summary of alert execution duration
+var FrontendMetrics map[string]FrontendMetricsRecorder = map[string]FrontendMetricsRecorder{}
+
+func registerFrontendSummary(name string, help string) {
+	objectiveMap := map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001}
+
+	summary := prometheus.NewSummary(prometheus.SummaryOpts{
```

Comment:
I think we should use a histogram instead. Summaries cannot be aggregated correctly. 
ex https://github.com/grafana/grafana/blob/master/pkg/middleware/request_metrics.go#L33 but with other buckets :)

Reply:
@bergquist ok was wondering why we where not using histogram for alert exec time and rendering time, so figured that was preferred over histogram for some reason. Will switch to histogram